packages/warriorjs-cli/templates/readme/abilities.ejs

```diff
@@ -1,17 +1,20 @@
 <%_
   const actions = abilities.filter(ability => ability.action);
   const senses = abilities.filter(ability => !ability.action);
+
+  const sortedActions = actions.sort((a, b) => a.name > b.name);
+  const sortedSenses = senses.sort((a, b) => a.name > b.name);
```

Comment:
Hey @djohalo2, what I originally meant is that you can sort the entire array of abilities just once, and then filter it. The resulting code is much simpler:

```js
const sortedAbilities = abilities.sort((a, b) => a.name > b.name);
const actions = sortedAbilities.filter(ability => ability.action);
const senses = sortedAbilities.filter(ability => !ability.action);
```